public/assets/js/scripts.js

```diff
@@ -146,4 +146,100 @@ $(document).ready(function () {
     });
   });
 
+  // use ajax to update select options on index page
+  const state = $("#state");
+  const area = $("#area");
+  const type = $("#type");
+
+  $.ajax({
+    type: 'POST',
+    url: './includes/find-house.php',
+    data: {
+      load_state: true
+    },
+    success: function(data){
+      data = JSON.parse(data);
+      var text = "<option selected disabled>State</option>";
+
+      for (let i = 0; i < data.length; i++) {
+        text += `<option value="${data[i].state_id}">${data[i].state_name}</option>`;
+      }
+
+      $(state).html(text);
```

Comment:
Can you verify this, please? I think this should be 
```suggestion
      state.html(text);
```